vumi_wikipedia/tests/test_wikipedia_api.py

```diff
@@ -116,6 +116,18 @@ def test_empty_input(self):
     resource_stream(__name__, 'wikipedia_responses.json'))
 
 
+def rewrite_request_line(request_line):
+    """
+    Sort the request parameters in the URL path so tests don't rely on
+    deterministic dict ordering.
+    """
+    method, sp, url_path = request_line.partition(' ')
+    path, q, params = url_path.partition('?')
+    params = '&'.join(sorted(params.split('&')))
+    url_path = q.join([path, params])
```

Comment:
`urlparse.urlparse()` doesn't really buy us anything (it just replaces the `url_path.partition()` line) and `urlparse.parse_qs` does a bunch of unescaping that we don't really want.